_posts/2023-11-19-Custom-JWT-validators-for-Sping-Security-starter.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title:  "Custom JWT validators for Spring Security starter"
+date:   2023-11-19 15:26:00 +0200
+categories: spring security
+description: Custom JWT validators for Spring Security starter
+---
+
+### Use case
+
+It's a common case when REST API is secured with OAuth 2.0 and acts as a Resouce server in terms of Auth 2.0. The easiest way to achieve this with Spring Security is to use [Spring Boot OAuth2 Resource Server starter](https://docs.spring.io/spring-security/reference/servlet/oauth2/resource-server/index.html){:target="_blank"}. This starter requires minimal configuration and works perfectly fine out of the box. You can see an example of the project in the official Spring repository with examples - [Spring Security Examples](https://github.com/spring-projects/spring-security-samples/blob/main/servlet/spring-boot/java/oauth2/resource-server/hello-security){:target="_blank"}.

```

Comment:
`{:target="_blank"}` Looks a bit strange. It's not parsed eventually. 

Also check other appearances of this tag.

Reply:
This is OK, works fine in the previous posts. It helps to open the links in a new tab https://talk.jekyllrb.com/t/how-can-i-ensure-my-links-open-in-a-new-tab/4318